libs/utils/utils.js

```diff
@@ -1049,7 +1049,7 @@ async function checkForPageMods() {
 
 async function loadPostLCP(config) {
   await decoratePlaceholders(document.body.querySelector('header'), config);
-  if (config.mep?.targetEnabled === 'gnav') {
+  if (config.mep?.targetEnabled === 'postlcp') {
```

Comment:
Will this break existing pages that are using `gnav`?

Reply:
we had one test where we used gnav and that test ended.  So we removed the setting from all pages.  This update should be in place long before the next test that needs this setting.